tests/functional/object/rgw/test_rgw_sts.py

```
@@ -877,4 +882,336 @@ def _is_token_expired():
+            "s3", endpoint, region, iam_user_access_key, iam_user_secret_key
+        )
+        sts_client = self.make_boto3_client(
+            "sts", endpoint, region, iam_user_access_key, iam_user_secret_key
+        )
+        role_name, role_arn, policy_name = self.create_iam_role(
+            iam_client, account_id, with_permission_policy=False
+        )
+
+        bucket_name = create_unique_resource_name("sts-test", "bucket")
+        iam_user_s3_client.create_bucket(Bucket=bucket_name)
+        logger.info(f"Bucket {bucket_name} created successfully")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)